cli/server/server.go

```diff
@@ -202,6 +206,9 @@ func dumpDB(ctx *cli.Context) error {
 	if count == 0 {
 		count = chainCount - start
 	}
+	if start != 0 {
+		writer.WriteU32LE(start)
```

Comment:
We should also add a note to `start` documentation as the output format differs with and without this option.

Reply:
Added to command documentation.